src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj

```diff
@@ -13,6 +13,7 @@
     </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="15.5.32" />

```

Comment:
is it necessary? Dependency management is a pain and whenever possible, we should avoid dependencies

Reply:
Yeah I know. Is there another implementation of an Async aware lock in the framework? I suppose I could lift out the implementation and embed it directly but that feels a bit dirty

Reply:
can SemaphoreSlim help here?

Reply:
I think so...checking this out.

Reply:
works great.